### ISSUE TITLE
[Bug] select-apps locks up on Windows

### ISSUE THREAD

**Issue by tpill90** [tpill90/battlenet-lancache-prefill#122]
See [SteamPrefill issues/366](https://github.com/tpill90/steam-lancache-prefill/issues/366)

**Comment by Roemer** [comment]
Any chance to get this fixed soon?

**Comment by tpill90** [comment]
Yeah, I'll get this pushed up in the next few days when I get a minute.

**Comment by tpill90** [comment]
Pushed up in [v2.1.0](https://github.com/tpill90/battlenet-lancache-prefill/releases/tag/v2.1.0)
